internal/cli/atlas/livemigration/link/create.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package link
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var createTemplate = "Link Token '{{.LinkToken}}' successfully created.\n"
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store         store.LinkTokenCreator
+	orgID         string
+	accessListIPs []string
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+func (opts *CreateOpts) Run() error {
+	createRequest := opts.newTokenCreateRequest()
+
+	r, err := opts.store.CreateLinkToken(opts.orgID, createRequest)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newTokenCreateRequest() *mongodbatlas.TokenCreateRequest {
+	return &mongodbatlas.TokenCreateRequest{
+		AccessListIPs: opts.accessListIPs,
+	}
+}
+
+// mongocli atlas liveMigration|lm link create [--accessListIps accessListIps] [--orgId orgId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create one new link-token.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.initStore,
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.orgID, flag.OrgID, "", usage.OrgID)
+	cmd.Flags().StringSliceVar(&opts.accessListIPs, flag.AccessListIPs, []string{}, usage.AccessListCIDREntries)
```

Comment:
I think I flagged this in the spec doc, [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html) recommends the use of singular, we have some legacy flags in plural for backward compatibility but we stick to singular ever since getting more familiar with POSIX standards